kie-dmn-core/src/test/java/org/kie/dmn/core/FlightRebookingTest.java

```diff
@@ -189,6 +214,21 @@ private List loadFlightList() {
         }
         return flightList;
     }
+    
+    /**
+     * Same as {@link #loadFlightList()} but instead of using Java LocalDateTime, using FEEL "date and time" datatype.
+     */
+    private List loadFlightListFEEL() {
+        Object[][] flightData = new Object[][] {
+                {"UA123", "SFO", "SNA", "date and time(\"2017-01-01T18:00:00\")", "date and time(\"2017-01-01T19:00:00\")", 5, "cancelled"},
+                {"UA456", "SFO", "SNA", "date and time(\"2017-01-01T19:00:00\")", "date and time(\"2017-01-01T20:00:00\")", 2, "scheduled"},
+                {"UA789", "SFO", "SNA", "date and time(\"2017-01-01T21:00:00\")", "date and time(\"2017-01-01T23:00:00\")", 2, "scheduled"},
+                {"UA1001", "SFO", "SNA", "date and time(\"2017-01-01T23:00:00\")", "date and time(\"2017-01-02T05:00:00\")", 0, "scheduled"},
+                {"UA1111", "SFO", "LAX", "date and time(\"2017-01-01T23:00:00\")", "date and time(\"2017-01-02T05:00:00\")", 2, "scheduled"}
+        };
+
+        return flightDataToFlightList(flightData);
+    }
```

Comment:
I am confused... where are you converting the strings into actual date/time objects?